lib/payload.json

```diff
@@ -137,6 +145,32 @@
             }
           },
           "additionalProperties": false
+        },
+        "water": {
+          "type": "object",
+          "properties": {
+            "depth": {
+              "description": "Depth of water measurement (cm)",
+              "$ref": "#/definitions/depth"
+            },
+            "temperature": {
+              "description": "Water temperature (°C)",
+              "$ref": "#/definitions/temperature"
+            },
+            "ec": {
+              "description": "Water electrical conductivity (dS/m)",
+              "$ref": "#/definitions/ec"
+            },
+            "pH": {
```

Comment:
I understand that we can always convert units easily, but indeed we decided to stay as close to the domain as possible.

My understanding is that `dS/m` is common in the soil domain, while here it might be `S/m`? Not sure. But we prefer using domain units over reusing units.

Reply:
I have removed the water ec while I look for what the unit should be. Google tells me people use anything from uS to S, cm to m. Different sensors may return different units.